images/hub/jupyterhub_config.py

```diff
@@ -136,6 +136,9 @@
     c.GitHubOAuthenticator.oauth_callback_url = get_config('auth.github.callback-url')
     c.GitHubOAuthenticator.client_id = get_config('auth.github.client-id')
     c.GitHubOAuthenticator.client_secret = get_config('auth.github.client-secret')
+    org_whitelist = get_config('auth.github.org_whitelist', [])
```

Comment:
you'll want to also add `auth.github.org_whitelist` to the configmap and the default values.yaml, so that it propagates all they way from user config to here